packages/grid/src/GridRenderer.ts

```diff
@@ -2503,6 +2557,109 @@ export class GridRenderer {
         handleWidth,
         hScrollBarSize - scrollBarCasingWidth
       );
+
+      if (!autoSelectRow && scrollBarSelectionTickColor != null) {
+        context.fillStyle = scrollBarSelectionTickColor;
+        // Scrollbar Selection Tick
+        const { selectedRanges, model, cursorColumn } = state;
+        const { visibleColumns, visibleColumnWidths } = metrics;
+        const { columnCount } = model;
+
+        const leftVisibleIndex = Math.min(...visibleColumns.values());
+        const rightVisibleIndex = Math.max(...visibleColumns.values());
+
+        const getTickX = (index: number) => {
```

Comment:
Something still seems off with this... looking at Tree Grid in the style guide for testing, and if all columns are the same width, then we shouldn't have the ticks moving around at all - they should be correctly and consistently positioned no matter where you scroll. It's only in cases where there's variable column sizes where it should move around a bit.

It may also be that the scroll bar thumb width isn't the "correct" size - the thumb within the scroll bar should match the viewport compared to the whole grid view.